### PR TITLE
Update telemetry_fields.yaml

### PR DESCRIPTION
**Pull request by RathishKJ**
Hello,

Request you to kindly add the below points to be added to the pull request.  This pull request is further to the issue request #502 

- return_air_isolation_damper_status
	- OPEN
	- CLOSE
- fan_run_time_accumulator
- compressor_run_time_accumulator
- evaporative_fan_speed_sensor
- condensing_fan_speed_percentage_command
- high_limit_zone_air_temperature_setpoint
- low_limit_zone_air_temperature_setpoint
- high_limit_zone_air_pressure_setpoint
- low_limit_zone_air_pressure_setpoint
- low_limit_zone_air_pressure_setpoint
- high_limit_zone_air_relative_humidity_setpoint
- low_limit_zone_air_relative_humidity_setpoint
- zone_air_override_temperature_command
	- ACTIVE
	- INACTIVE

regards,
Rathish

### CODE REVIEW

**Comment by vasmys** [comment]
> Hello,
> 
> Request you to kindly add the below points to be added to the pull request. This pull request is further to the issue request #502
> 
>     * return_air_isolation_damper_status
>       
>       * OPEN
>       * CLOSE
> 
>     * fan_run_time_accumulator
> 
>     * compressor_run_time_accumulator
> 
>     * evaporative_fan_speed_sensor
> 
>     * condensing_fan_speed_percentage_command
> 
>     * high_limit_zone_air_temperature_setpoint
> 
>     * low_limit_zone_air_temperature_setpoint
> 
>     * high_limit_zone_air_pressure_setpoint
> 
>     * low_limit_zone_air_pressure_setpoint
> 
>     * low_limit_zone_air_pressure_setpoint
> 
>     * high_limit_zone_air_relative_humidity_setpoint
> 
>     * low_limit_zone_air_relative_humidity_setpoint
> 
>     * zone_air_override_temperature_command
>       
>       * ACTIVE
>       * INACTIVE
> 
> 

Hello,

The below points are deleted since the points are existing in other pull requests

•	return_air_isolation_damper_status 
  o	OPEN
  o	CLOSE
•	fan_run_time_accumulator
•	compressor_run_time_accumulator
•	evaporative_fan_speed_sensor
•	condensing_fan_speed_percentage_command
•	high_limit_zone_air_temperature_setpoint
•	low_limit_zone_air_temperature_setpoint
•	high_limit_zone_air_pressure_setpoint
•	low_limit_zone_air_pressure_setpoint
•	high_limit_zone_air_relative_humidity_setpoint
•	low_limit_zone_air_relative_humidity_setpoint

Regards,
rathish

**Comment by RathishKJ** [comment]
> Hello,
> 
> Request you to kindly add the below points to be added to the pull request. This pull request is further to the issue request #502
> 
>     * return_air_isolation_damper_status
>       
>       * OPEN
>       * CLOSE
> 
>     * fan_run_time_accumulator
> 
>     * compressor_run_time_accumulator
> 
>     * evaporative_fan_speed_sensor
> 
>     * condensing_fan_speed_percentage_command
> 
>     * high_limit_zone_air_temperature_setpoint
> 
>     * low_limit_zone_air_temperature_setpoint
> 
>     * high_limit_zone_air_pressure_setpoint
> 
>     * low_limit_zone_air_pressure_setpoint
> 
>     * low_limit_zone_air_pressure_setpoint
> 
>     * high_limit_zone_air_relative_humidity_setpoint
> 
>     * low_limit_zone_air_relative_humidity_setpoint
> 
>     * zone_air_override_temperature_command
>       
>       * ACTIVE
>       * INACTIVE
> 
> 
> regards, Rathish

Hello,

The below points are deleted since the points are existing in other pull requests

• return_air_isolation_damper_status
o OPEN
o CLOSE
• fan_run_time_accumulator
• compressor_run_time_accumulator
• evaporative_fan_speed_sensor
• condensing_fan_speed_percentage_command
• high_limit_zone_air_temperature_setpoint
• low_limit_zone_air_temperature_setpoint
• high_limit_zone_air_pressure_setpoint
• low_limit_zone_air_pressure_setpoint
• high_limit_zone_air_relative_humidity_setpoint
• low_limit_zone_air_relative_humidity_setpoint

Regards,
rathish

**Comment by tasodorff** [comment]
This duplicates another PR. Please close.

**Comment by tasodorff** [comment]
@RathishKJ please close and consolidate with other PR.

**Comment by RathishKJ** [comment]
> @RathishKJ please close and consolidate with other PR.

the rest of the points in this PR had been deleted except **zone_air_override_temperature_command**
The syntax issue also been resolved...

File: ontology/yaml/resources/fields/telemetry_fields.yaml [hunk 1]
```diff
@@ -1160,5 +1160,14 @@ literals:
 - return_air_specificenthalpy_sensor
 - compressor_current_sensor
 
+## New Fields For ACUs
+- zone_air_override_temperature_command
```
Comment (tasodorff) [inline ontology/yaml/resources/fields/telemetry_fields.yaml:1164]: what is this and why is it needed? was not resolved in [issue 502](https://github.com/google/digitalbuildings/issues/502) and i still do not understand how it works, or why it is required. i am also not convinced it is a command.

**Review by tasodorff — COMMENTED** [review]
Please clarify the necessity of this field. If we see the zone temp setpoints, then what is the use of this field?

**Comment by RathishKJ** [comment]
deleting the PR since the requested telemetries in this current PR have been raised in the respective Devices' PRs

**Pull request closed without merging**